tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -13,21 +15,35 @@
 from rest_framework import status
 from rest_framework.authtoken.views import ObtainAuthToken
 from rest_framework.response import Response
-from typing import Dict, Optional
 
-from .login_redirect_oidc import LoginRedirectAMS
 from ..authentication import CustomAuthentication
+from .login_redirect_oidc import LoginRedirectAMS
 from .utils import (
-    generate_token_endpoint_parameters,
+    generate_client_assertion,
     generate_jwt_from_jwks,
-    validate_nonce_and_state,
+    generate_token_endpoint_parameters,
     response_redirect,
-    generate_client_assertion,
+    validate_nonce_and_state,
 )
 
 logger = logging.getLogger(__name__)
 
 
+def print_better(message, table):
```

Comment:
wrote this trying to log a lot of objects. It shouldn't stay here, but we could put it in utils. I thought this was useful